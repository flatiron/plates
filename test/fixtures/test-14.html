<table>
  <tr class="names">
    <td class="first"></td>
    <td class="last"></td>
  </tr>
</table>
