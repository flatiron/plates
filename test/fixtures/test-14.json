{
  "names" : [
    {
      "first"  : "John",
      "last"   : "Smith"
    },
    {
      "first"  : "Jane",
      "last"   : "Doe"
    }
  ]
}
