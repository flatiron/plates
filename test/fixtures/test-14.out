<table>
  <tr class="names">
    <td class="first">John</td>
    <td class="last">Smith</td>
  </tr><tr class="names">
    <td class="first">Jane</td>
    <td class="last">Doe</td>
  </tr>
</table>
